### PR TITLE
EUI 2646 & 3024 & EUI-3105 & EUI-2569- bug fixes related to searching cases and viewing case details

### PR DESCRIPTION
**Pull request by leepavlouhmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2646
https://tools.hmcts.net/jira/browse/EUI-3024
https://tools.hmcts.net/jira/browse/EUI-3105
https://tools.hmcts.net/jira/browse/EUI-2569

### Change description ###
Made some minor changes to facilitate the changes to the npm package eg when the reset button is clicked on the case search page as well as the "any" option on casestate field

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "@angular/platform-browser": "^7.2.13",
     "@angular/platform-browser-dynamic": "^7.2.13",
     "@angular/router": "^7.2.13",
-    "@hmcts/ccd-case-ui-toolkit": "^2.67.3-tab-name-replaceAll",
+    "@hmcts/ccd-case-ui-toolkit": "^2.68.9-alpha",
```
Comment (udaydenduluri33) [inline package.json:68]: Are you sure about releasing alpha to master?

Comment (Mo-Lala84) [reply package.json:68]: ii agree however until ronald comes back with the ways of semantic versioning :-)..this release for live defects is well overdue and we cant hold it back unfortunately.

**Comment by rayliangatos** [comment]
> Kudos, SonarCloud Quality Gate passed!
> 
> [Bug](https://sonarcloud.io/project/issues?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=BUG)
> [Vulnerability](https://sonarcloud.io/project/issues?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=VULNERABILITY)
> [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=SECURITY_HOTSPOT)
> [Code Smell](https://sonarcloud.io/project/issues?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=CODE_SMELL) [0 Code Smells](https://sonarcloud.io/project/issues?id=rpx-xui-webapp&pullRequest=850&resolved=false&types=CODE_SMELL)
> 
> [](https://sonarcloud.io/component_measures?id=rpx-xui-webapp&pullRequest=850&metric=new_coverage&view=list) [73.7% Coverage](https://sonarcloud.io/component_measures?id=rpx-xui-webapp&pullRequest=850&metric=new_coverage&view=list)
> [](https://sonarcloud.io/component_measures?id=rpx-xui-webapp&pullRequest=850&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=rpx-xui-webapp&pullRequest=850&metric=new_duplicated_lines_density&view=list)

Test coverage rate < 80%, it probably can't pass the CI.

File: src/cases/containers/case-list/case-list.component.ts [hunk 2]
```diff
@@ -309,7 +309,7 @@ export class CaseListComponent implements OnInit, OnDestroy {
         caseState,
         metadataFields,
         formGroup: {
-          value: formGroupValues
+          value: (formGroupValues ? formGroupValues : {})
```
Comment (rayliangatos) [inline src/cases/containers/case-list/case-list.component.ts:312]: the () can be removed?

Comment (leepavlouhmcts) [reply src/cases/containers/case-list/case-list.component.ts]: removed

File: package.json (hunk 1 above)
Comment (udaydenduluri33) [reply package.json:68]: ok

File: api/configuration/references.ts [hunk 3]
```diff
@@ -28,7 +28,7 @@ export const SERVICES_CCD_DATA_STORE_API_PATH = 'services.ccd.dataApi'
 export const SERVICES_CCD_CASE_ASSIGNMENT_API_PATH = 'services.ccd.caseAssignmentApi'
 
 export const SERVICES_DOCUMENTS_API_PATH = 'services.documents.api'
-
+export const SERVICES_FEE_AND_PAY_API_PATH = 'services.feeAndPayApi'
```
Comment (udaydenduluri33) [inline api/configuration/references.ts:31]: Do we need this? We not using feeAndPayApi

File: package.json (hunk 1 above)
Comment (udaydenduluri33) [reply package.json:68]: Ok

File: api/configuration/references.ts (hunk 3 above)
Comment (leepavlouhmcts) [reply api/configuration/references.ts:31]: i dont think we do. this was from the health check endpoint ticket merge. i did the rebase wrong yesterday, so its popped into this branch.

Comment (udaydenduluri33) [reply api/configuration/references.ts:31]: please remove it. It will be an issue.

Comment (leepavlouhmcts) [reply api/configuration/references.ts]: its removed

**Review by udaydenduluri33 — APPROVED** [review]
LGTMl

**Pull request merged**